indico/modules/events/registration/templates/emails/invitation_default_body.html

```diff
@@ -0,0 +1,29 @@
+{% extends 'emails/base.html' %}
+
+{% block header_recipient -%}
+    {first_name}
+{%- endblock %}
+
+{% block body %}
+    {% block body_text %}
+        We'd like to invite you to register for the following event:<br>
+        <strong>{{ event.title }}</strong><br><br>
+
+        You can register (or decline the invitation) by using this link:<br>
+        {invitation_link}
```

Comment:
yes, this is a placeholder, not a jinja var. the text is customizable by the user